tests/queries/tests.py

```diff
@@ -440,6 +440,14 @@ def test_ticket2496(self):
             ['<Item: four>']
         )
 
+    def test_ticket24277(self):
+        kws = {'note': 'n1', 'misc': 'foo'}
+        self.assertRaisesMessage(
+            FieldError,
+            'Cannot parse keyword query as dict',
+            lambda: Note.objects.filter(kws)
```

Comment:
You can use `assertRaisesMessage(FieldError, '...', Note.objects.filter, kws)` instead of the `lambda` here.
I think it'd improve readability a bit.